src/test/scala/is/hail/methods/LinearRegressionMultiPhenoSuite.scala

```diff
@@ -121,7 +121,7 @@ class LinearRegressionMultiPhenoSuite extends SparkSuite {
         "Sample",
         root = Some("sa.pheno"),
         config = TextTableConfiguration(types = Map("Pheno" -> TDouble), missing = "0"))
-      .linregMultiPheno(Array("sa.pheno.Pheno"), Array("sa.cov.Cov1", "sa.cov.Cov2 + 1 - 1"), "va.linreg", useDosages = true, 1, 0.0)
+      .linregMultiPheno(Array("sa.pheno.Pheno"), Array("sa.cov.Cov1", "sa.cov.Cov2"), useDosages = true)
```

Comment:
You're also converting an expression to an annotation path, was that intentional?